src/gui/widgets/text_box.cpp

```diff
@@ -400,6 +404,16 @@ text_box_definition::text_box_definition(const config& cfg)
 	load_resolutions<resolution>(cfg);
 }
 
+/**
+ * @ingroup GUIWidgetDefinitionWML
+ *
+ * The resolution for a text box also contains the following keys:
+ * Key          |Type                                    |Default  |Description
+ * -------------|----------------------------------------|---------|-----------
+ * text_x_offset| @ref guivartype_f_unsigned "f_unsigned"|""       |The x offset of the text in the text box. This is needed for the code to determine where in the text the mouse clicks, so it can set the cursor properly.
+ * text_y_offset| @ref guivartype_f_unsigned "f_unsigned"|""       |The y offset of the text in the text box.
```

Comment:
The CI build will fail if rebuilt with the move-docs-to-Wiki change already merged, because Doxygen no longer knows what a `guivartype_f_unsigned` is.

Reply:
Noticed, removing it.